drivers/audio/Kconfig.wm8960

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2021, NXP
+# SPDX-License-Identifier: Apache-2.0
+
+config AUDIO_WM8960
+	bool "use wm8960"
+	select I2C
+	help
+	  This option is to use the WM8960 codec
+
+if AUDIO_WM8960
+
+config AUDIO_LOOP_DIVIDER
+	int "PLL loop divider"
```

Comment:
Is this config used by the driver?

Reply:
remove then, which is not configurable for now.